pkg/ttnpb/udp/packet_data.go

```diff
@@ -53,6 +53,20 @@ type RxPacket struct {
 	Aesk  uint         `json:"aesk"`            // AES key index used for encrypting fine timestamps (unsigned integer)
 }
 
+// normalizeCodingRate normalizes the coding rate fraction returned by the packet forwarder.
+func normalizeCodingRate(cr string) string {
+	switch cr {
+	case "4/6":
+		return "2/3"
+	case "4/8":
+		return "1/2"
+	case "2/6":
+		return "1/3"
+	default:
+		return cr
+	}
+}
```

Comment:
If this is generally speaking always the same, why is `4/8` even a notation?

Note that this breaks TOA if you don't incorporate this in `pkg/toa`.

Reply:
Settled offline - the reasoning behind this `4/x` notation has been added to the function documentation.